src/i18n/strings/en_EN.json

```diff
@@ -407,7 +407,14 @@
         "download_logs": "Download logs",
         "downloading_logs": "Downloading logs",
         "error_empty": "Please tell us what went wrong or, better, create a GitHub issue that describes the problem.",
-        "failed_send_logs": "Failed to send logs: ",
+        "failed_send_logs": "Failed to send logs. Could not contact the rageshake server.",
```

Comment:
Reusing the i18n key will be confusing here as the meaning has drastically changed. Also doesn't this fit within "causes"? e.g. cause=unreachable

Reply:
The previous string (which will be what all other languages use for the foreseeable) is a partial sentence, ending with a colon. It'll never have any text rendered after it again, it'll look broken. It still needs a new key

Reply:
Okay, round 3 (4?). Tried to tidy this up.